model/src/main/scala/org/broadinstitute/dsde/rawls/model/Notification.scala

```diff
@@ -67,16 +70,17 @@ object Notifications {
   val ActivationNotificationType = register(new NotificationType[ActivationNotification] {
     override val format = jsonFormat1(ActivationNotification.apply)
     override val description = "Account Activation"
+    override val alwaysOn = true
```

Comment:
Since this is where we are defining the business logic as to whether a notification is disable-able, let's also include that information in the JSON response so the UI can accurately determine which ones to prevent disabling.